python/py-gnupg/Portfile

```diff
@@ -24,15 +23,12 @@ platforms            darwin
 maintainers          {@F30 f30.me:f30} openmaintainer
 
 python.versions      27 34 35 36
-depends_run          path:bin/gpg:gnupg2
+depends_run          bin:gpg:gnupg2
```

Comment:
Why? This allows a gpg binary installed outside of MacPorts to satisfy the dependency, and in almost all situations, we do not want that.

Reply:
Okay, so then @F30 could you submit a pull to reverse this please?

Reply:
@ryandesign Well, personally I want that, but if MacPorts policy says otherwise so be it. However, afaict the actual binary getting used will depend on the user's `PATH`, so we can't be sure its the one from MacPorts anyway.

Reply:
Almost everyone using MacPorts will have the PATH setup the way that MacPorts sets it up, so they would get the gpg MacPorts installs. 

Having a `gpg` binary of unknown vintage and parentage being used has the potential to generate issues that would be totally impossible to debug. It's a matter of consistency, to allow the chance of support, at least. 

People are certainly free to modify their own port trees to suit themselves, of course. I do it all the time, for things I want a certain way on my system. It is very easy to do that, so if you want something special, you can certainly have it for yourself!

Reply:
Runtime dependencies which only call an external/independent binary are in my opinion less problematic than linking against random libraries. GnuGP does provide an official installer. I don't know how heavy dependencies the port itself is introducing. I would certainly be in favour of depending on the port, but given that it's just a runtime dependency I would not be in such a strong opposition of allowing external binary in this case.

No clue what the benefits of installing a standalone gpg binary are though, and there's a big chance that another port would pull in the dependency anyway.